### PR TITLE
Upgrade to 1.7.1

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

**Comment by frol** [comment]
@jakirkham All tests are green. It is ready to merge.

**Comment by frol** [comment]
@jakirkham Ping

**Comment by jakirkham** [comment]
Thanks @frol. Could you please re-render as well?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,10 +25,11 @@ requirements:
...
     - python
     - setuptools
+    - pytest-runner
 
   run:
     - python
-    - astroid >=1.4.5,<1.5.0
+    - astroid >=1.5.1,<1.6.0
     - backports.functools_lru_cache  # [py27]
     - configparser                   # [py27]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Appears they also use [`singledispatch` for `py2k`]( https://github.com/PyCQA/pylint/blob/pylint-1.7.1/pylint/__pkginfo__.py#L48 ). Probably should make these selectors `py2k` too.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,10 +25,11 @@ requirements:
   build:
     - python
     - setuptools
+    - pytest-runner
 
   run:
     - python
-    - astroid >=1.4.5,<1.5.0
+    - astroid >=1.5.1,<1.6.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Doesn't appear to be [upper bounded]( https://github.com/PyCQA/pylint/blob/pylint-1.7.1/pylint/__pkginfo__.py#L25 ).

Comment (frol) [inline recipe/meta.yaml]: Resolved.

File: recipe/meta.yaml (hunk 1 above)
Comment (frol) [inline recipe/meta.yaml]: Resolved.

**Comment by jakirkham** [comment]
Thanks @frol.

**Pull request merged**